mac/expert/determine_processor_architecture.tw2

```diff
@@ -0,0 +1,32 @@
+::Determine processor architecture on a Mac - Expert
+# Determine processor architecture on a Mac
```

Comment:
For the Experts path I'm not sure we need to have them verify their processor architecture. Or if we do, then we can just say `uname -m` rather than having the if/else. We could also say that it's probably 64-bit.

Reply:
I agree with removing the if/else afterwards and also saying about the 64-bit probably. 

I think is quite handy to have the `uname -m` command easily reachable through the walkthrough, I personally don't know by heart the command and someone that doesn't know his architecture must probably too.

Reply:
> I think is quite handy to have the uname -m command easily reachable through the walkthrough

Agreed!